packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -510,6 +511,11 @@ export default class TabAdvanced extends Component {
   _onChangeCpuMask = cpuMask =>
     editVm(this.props.vm, { cpuMask: map(cpuMask, 'value') })
 
+  _handleBootFirmware = value =>
+    editVm(this.props.vm, {
+      hvmBootFirmware: value,
```

Comment:
What if the value is an empty string?

Reply:
I coded the thing to not be empty, but I fixed it anyway :heavy_check_mark: